crates/node/gossip/src/gater.rs

```diff
@@ -181,12 +178,8 @@ impl ConnectionGate for ConnectionGater {
             return Err(DialError::ThresholdReached { addr: addr.clone() });
         }
 
-        // If the peer is blocked, do not dial.
-        if self.blocked_peers.contains(&peer_id) {
-            debug!(target: "gossip", peer=?addr, "Peer is blocked, not dialing");
-            kona_macros::inc!(gauge, crate::Metrics::DIAL_PEER_ERROR, "type" => "blocked_peer", "peer" => peer_id.to_string());
-            return Err(DialError::PeerBlocked { peer_id });
-        }
+        // Note: Peer blocking is now handled by libp2p's allow_block_list behaviour
+        // and does not need to be checked here.
```

Comment:
This won't work well now since we will tell connection gate users that they can dial a peer when they really cant